controllers/hostedcontrolplane/hostedcontrolplane.go

```diff
@@ -222,39 +226,6 @@ func (r *HostedControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.R
 	return ctrl.Result{}, err
 }
 
-// isVpcEndpointReady checks if the VPC Endpoint associated with the HostedControlPlane is ready.
-func (r *HostedControlPlaneReconciler) isVpcEndpointReady(ctx context.Context, hostedcontrolplane *hypershiftv1beta1.HostedControlPlane) (bool, error) {
```

Comment:
Instead of moving this entire function to a different file can't you just rename to `IsVpcEndpointReady` so it can be used by the other file?

Reply:
This isn't actually a functional change - more of an organizational one since the `isVpcEndpointReady()` function seemed more appropriate in the healthcheck.go file :slightly_smiling_face: